modelon/impact/client/entities/case.py

```diff
@@ -503,12 +487,9 @@ def get_trajectories(self) -> Result:
         """Returns result(Mapping) object containing the result trajectories.
 
         Returns:
-
-            trajectories:
-                A result trajectory dictionary object.
+            A result trajectory dictionary object.
```

Comment:
I know this was just formatting change now but the comment is misleading since this is not a "dictionary". I suggest:

A Result object (mapping) that allows requesting trajectory data given a variable name.